command/sub8_missions/missions/bin_dropper.py

```diff
@@ -0,0 +1,60 @@
+from twisted.internet import defer
+from txros import util, tf
+import numpy as np
+
+
+@util.cancellableInlineCallbacks
+def run(sub):
+    # bin_search is a Deferred
+    print "We're looking for a buoy"
+
+    print "Executing search pattern"
+    #yield sub.move.right(2.0).go()
+    #yield sub.move.down(0.3).go()
+
+    print "Now trying to pose it"
+    response = yield sub.bin.get_pose('norange')
+    if not response.found:
+        print 'failed to discover bin location'
+        return
+    else:
+        print "Got bin pose"
+        print response.pose
+    while True:
+
+        response = yield sub.bin.get_pose('norange')
+        print "Bin at", response.pose
+        if abs(response.pose.x)>10 or abs(response.pose.y)>10:
+            if response.pose.x<0:
```

Comment:
Thanks, I used left and right for readability. I see how + and - would look cleaner.